src/Main.java

```diff
@@ -1,5 +1,71 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        // Task 1
+        var dog = 8.0;
+        var cat = 3.6;
+        var paper = 763789;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+
+        // Task 2
+        dog += 4;
+        cat += 4;
+        paper += 4;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+
+        // Task 3
+        dog -= 4;
+        cat -= 1.6;
+        paper -= 7639;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+
+        // Task 4
+        var friend = 19;
+        System.out.println(friend);
+        friend += 2;
+        System.out.println(friend);
+        friend /= 7;
+        System.out.println(friend);
+
+        // Task 5
+        var frog = 3.5;
+        System.out.println(frog);
+        frog *= 10;
+        System.out.println(frog);
+        frog /= 3.5;
+        System.out.println(frog);
+        frog += 4;
+        System.out.println(frog);
+
+        // Task 6
+        var firstSportsmen = 78.2;
+        var secondSportsmen = 82.7;
+        var weightSum = firstSportsmen + secondSportsmen;
+        var weightDiff = Math.abs(firstSportsmen - secondSportsmen);
+        System.out.println(weightSum);
+        System.out.println(weightDiff);
+
+        // Task 7
+        var weightDiffMethod1 = secondSportsmen - firstSportsmen;
+        var weightDiffMethod2 = secondSportsmen % firstSportsmen;
+        System.out.println(weightDiffMethod1);
+        System.out.println(weightDiffMethod2);
+
+        // Task 8
+        var timeOfWork = 640;
+        var everyEmployerTime = 8;
+        var amountOfEmployers = timeOfWork / everyEmployerTime;
+        System.out.println("Всего работников в компании — " + amountOfEmployers);
+
+        amountOfEmployers += 94;
+        timeOfWork = amountOfEmployers * 8;
```

Comment:
```suggestion
        timeOfWork = amountOfEmployers * everyEmployerTime;
```
переменные используем